@types/jsb.d.ts

```diff
@@ -358,1179 +358,413 @@ declare namespace jsb {
-         *
-         * @zh
-         * 新计划立即生效，新价格将在下次复诊时收取。
+         * @en Returns a Task which asynchronously unlocks an achievement for the currently signed in player.
+         * @zh 返回一个 Task 异步解锁当前登录玩家的成就。
          */
-        WITHOUT_PRORATION = 3,
+        public unlockImmediate(id: string): PlayTask;
         /**
-         * @en
-         * Replacement takes effect immediately, and the user is charged full price of new plan and
-         * is given a full billing cycle of subscription, plus remaining prorated time from the old plan.
-         *
-         * @zh
-         * 替换立即生效，用户将被收取新计划的全额费用，并获得一个完整的计费周期，加上旧计划按比例计算的剩余时间。
+        * @en Increments an achievement by the given number of steps.
+        * @zh 按给定的步数增加成就。
+        */
+        public increment(id: string, numSteps: number): void;
+        /**
+         * @en Reveals a hidden achievement to the currently signed-in player.
+         * @zh 向当前登录的玩家揭示隐藏的成就。
+         */
+        public reveal(id: string): void;
+        /**
+         * @en Sets an achievement to have at least the given number of steps completed.
+         * @zh 设置一项成就，至少完成给定数量的步骤。
          */
-        CHARGE_FULL_PRICE = 5,
+        public setSteps(id: string, numSteps: number): void;
         /**
-         * @en
-         * The new purchase takes effect immediately, the new plan will take effect when the old item expires.
-         *
-         * @zh
-         * 新购买立即生效，新计划将在旧项目到期时生效。
+         * @en Unlocks an achievement for the currently signed in player.
+         * @zh 为当前登录的玩家解锁一项成就。
          */
-        DEFERRED = 6,
+        public unlock(id: string): void;
     }
-
     /**
-     * @en
-     * Possible response codes.
-     *
-     * @zh
-     * 可能的响应代码。
+     * @en A client for the recall functionality.
+     * @zh Recall 功能客户端。
      */
-    export enum InAppMessageResponseCode {
-        /**
-         * @en
-         * The flow has finished and there is no action needed from developers.
-         *
-         * @zh
-         * 流程已经结束，开发人员无需采取任何行动。
-         */
-        NO_ACTION_NEEDED = 0,
+    export class RecallClient {
         /**
-         * @en
-         * The subscription status changed.
-         *
-         * @zh
-         * 订阅状态已更改。
+         * @en Returns a RecallAccess to use for server-to-server communication between the 3p game server and Play Games Services server.
+         * @zh 返回 RecallAccess 用于第三方游戏服务器和 Play 游戏服务服务器之间的服务器到服务器通信。
          */
-        SUBSCRIPTION_STATUS_UPDATED = 1
+        public requestRecallAccess(): PlayTask;
     }
 
     /**
-     * @en
-     * Recurrence mode of the pricing phase.
-     *
-     * @zh
-     * 定价阶段的复现模式。
+     * @en Main entry point for the Games APIs. This class provides APIs and interfaces to access the Google Play Games Services functionality.
+     * @zh 游戏 API 的主要入口点。此类提供用于访问 Google Play Games Services 功能的 API 和接口。
      */
-    export enum RecurrenceMode {
+    export class PlayGames {
```

Comment:
Doesn't `PlayGames` need to be removed in jsb.d.ts since it has already been defined in `vendor/google/index.ts?`

Reply:
I defined a class PlayGamesHelper in game.ts, which calls jsb.PlayGames. and PlayGamesHelper is the actual exported class, so there may be two